script/DeployViewer.s.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD 3-Clause License
```

Comment:
call the file

deployYnViewer.s.sol

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/119/commits/8aa1f28d964ab25d4fe416190fb37fb37d8fc98a